### PR TITLE
[AIRFLOW-6007] Check providers instead of provider package.

### PR DESCRIPTION
**Pull request by potiuk**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-6007

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Provider was specified instad of providers in the check for missing operators. This fixes it.

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

**Comment by potiuk** [comment]
@mik-laj - seems we are not checking providers now.

**Comment by mik-laj** [comment]
Are these dependencies needed? It seems to me that this change can be merged to the project without others.

**Comment by potiuk** [comment]
Without those changes, we have cyclic dependencies in this PR.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -217,12 +217,14 @@ repos:
         files: \.py$
         exclude: ^tests/.*\.py$|^airflow/_vendor/.*$
         pass_filenames: true
+        require_serial: true
```
Comment (feluelle) [inline .pre-commit-config.yaml]: Why do we want to `require_serial` ?

Comment (feluelle) [reply .pre-commit-config.yaml]: Also this change belongs to [AIRFLOW-6004], right?

**Review by feluelle — COMMENTED** [review]
Changes related to AIRFLOW-6007 lgtm.

**Comment by potiuk** [comment]
@feluelle @mik-laj -> This should be fine for review. Hopefully we merge the "executor" changes soon and we can merge this one as well.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (potiuk) [reply .pre-commit-config.yaml]: Yep.

**Comment by potiuk** [comment]
@mik-laj -> all tests passed and it's a standalone PR now. Can you please review?

**Review by mik-laj — APPROVED** [review]

**Pull request merged**